docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -15,12 +15,15 @@ The following table lists the available UI components:
 | ------------------------------------ | ---------------------------------------- | ---------------------------------------- |
 | [AlertDialog](./nui-components/AlertDialog.md)  | AlertDialog class shows a dialog with title, message, and action buttons. <br>[Guide ](./nui-components/AlertDialog.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.AlertDialog.html)  |  <img src="./nui-components/media/AlertDialog.png" width="600"> <br>[C# code for this page](./codeExamples/AlertDialog.cs) / [XAML page](./codeExamples/AlertDialog.xaml)                |
 | [Button](./nui-components/Button.md) | Button is a common component and describes what action occurs when you select it. A Button can either contain a text or an icon. <br>[Guide ](./nui-components/Button.md) / [API](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.Components.Button.html) | <img src="./nui-components/media/Button.png" width="300"> <br>[C# code for this page](codeExamples/Button.cs) / [XAML page](codeExamples/Button.xaml)               |
+| [CanvasView](./vectorgraphics/CanvasView.md) | A CanvasView is a class for displaying vector primitives. | `CanvasViewView` |
+| [CameraView](./nui-components/CameraView.md) | A camera view that controls and displays camera playback. | `CameraView` |
```

Comment:
@patrykka Thank you for making the requested additions, but the appearance section isn't like the other rows, as its missing an image, link to C# code, and link to XAML page.